doc/services/pm/device.rst

```diff
@@ -56,20 +56,30 @@ suspended.
    As functions in this context cannot block, transitions that intend to use blocking
    APIs **must** check whether they can do so with :c:func:`k_can_yield`.
 
-This type of device power management can be useful when the application is not
-power aware and does not implement runtime device power management. Though,
-:ref:`Device Runtime Power Management <pm-device-runtime-pm>` is the **preferred**
-option for device power management.
+This method of device power management can be useful in the following scenarios:
+
+ - Simple devices that do not require any blocking operation when suspending and
+   resuming.  This implementation is reasonably simpler than device runtime
+   power management.
```

Comment:
Talking about devices in this method makes things confused because the method applies to all devices in the system.

I would rather state: `Systems with no device requiring any blocking ...`

What if you have a mix of devices, some with blocking operation, some w/o blocking operation ?

Reply:
That is what I meant (and failed to clarify :P). It is indeed clear your proposal